examples/ct/structures_test.go

```diff
@@ -30,6 +30,17 @@ func TestGetCTLogID(t *testing.T) {
 	got64 := base64.StdEncoding.EncodeToString(got[:])
 
 	if expected != got64 {
-		t.Fatalf("Expected logID %s but got %s", expected, got64)
+		t.Fatalf("expected logID: %s but got: %s", expected, got64)
 	}
-}
\ No newline at end of file
+}
+
+func TestGetCTLogIDNotLoaded(t *testing.T) {
+	km := crypto.NewPEMKeyManager()
+
+	_, err := GetCTLogID(km)
+
+	assert.Error(t, err, "expected error when no key loaded")
+}
+
+func TestSignV1SCTForCertificate(t *testing.T) {
```

Comment:
Possible misunderstanding here. assert.Error checks that the function returned an error.